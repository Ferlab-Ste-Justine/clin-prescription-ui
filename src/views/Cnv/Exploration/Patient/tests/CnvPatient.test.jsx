import React from 'react';
import { Provider } from 'react-redux';
import { BrowserRouter as Router } from 'react-router-dom';
import renderer from 'react-test-renderer';
import configureStore from 'redux-mock-store';

import CnvPatient from '../index';

describe('SnvPatient', () => {
  test('should be as the snapshot', () => {
    const initialState = {};
    const store = configureStore()(initialState);
    const component = renderer.create(
      <Provider store={store}>
        <Router>
          <CnvPatient />
        </Router>
      </Provider>,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
