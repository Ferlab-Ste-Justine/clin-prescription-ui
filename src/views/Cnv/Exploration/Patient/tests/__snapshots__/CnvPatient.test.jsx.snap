// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SnvPatient should be as the snapshot 1`] = `null`;
